src/components/Publish/Navigation/index.tsx

```diff
@@ -1,16 +1,12 @@
 import { FormikContextType, useFormikContext } from 'formik'
 import React, { ReactElement } from 'react'
-import { FormPublishData } from '../_types'
+import { FormPublishData, StepContent } from '../_types'
```

Comment:
why is `StepContent` newly imported here?

Btw, are you using VSCode with our suggested plugins? Then you should see live warnings about ESlint failures. There really is no need to only solve unnecessary imports later during PR review

<img width="863" alt="Screen Shot 2022-01-25 at 09 55 20" src="https://user-images.githubusercontent.com/90316/150954229-29fb0def-a775-40d2-a68d-a8dbf85a3ad9.png">

Reply:
it's my fault beacuse I forgot to npm run lint